bluepysnap/circuit_validation.py

```diff
@@ -233,18 +233,18 @@ def _check_rotations():
     _check_rotations()
     components = config['components']
     errors += _check_components_dir('morphologies_dir', components)
-    errors += _check_components_dir('mechanisms_dir', components)
     errors += _check_components_dir('biophysical_neuron_models_dir', components)
     if errors:
         return errors
+    holder = group['@library'] if '@library' in group else group
```

Comment:
nice !

Reply:
Actually not :) It didn't work out well :(

Reply:
It's good to have tests

Reply:
hahaha tests are better than I am...